import/README.md

```diff
@@ -0,0 +1,80 @@
+# Importing data from Oracle to Postgres
+
+We would like to use PostgreSQL on cloud.gov for our rapid prototyping work on
+the NRM G&A application. The current production data is in NRM's Oracle
+database, so we need to copy data with the correct structure from Oracle and
+import it into our Postgres database. This is complicated by the Oracle server
+only being accessible on NRM's VPN. Here is what works for us, kind of.
+
+## Export from Oracle as TSV
+
+Using a SQL client, export a single G&A table as a TSV file. 
+
+The open-source (SQLWorkbench/J)[https://www.sql-workbench.eu/] is java-based
+and can be run most anywhere, on GSA and Forest Service laptops. It can create
+compressed TSV exports of tables using a command `WbExport`:
+
+```
+WbExport -type=text -compress
+-file='C:\Users\nmartinsenburrell\documents\notes.tsv'
```

Comment:
do you want to make the path more generic or anonymous?

Reply:
or note the reader of this will have to change it

Reply:
yes, done